tests/foreman/api/test_convert2rhel.py

```diff
@@ -412,5 +416,6 @@ def test_convert2rhel_centos_with_pre_conversion_template_check(
     centos.wait_for_connection()
 
     # Verify convert2rhel facts are generated, and verify fact conversions.success is true
-    assert centos.execute('test -f /etc/rhsm/facts/convert2rhel.facts').status == 0
     assert host_content['facts']['conversions::success'] == 'true'
+    convert2rhel_facts = json.loads(centos.execute('cat /etc/rhsm/facts/convert2rhel.facts').stdout)
+    assert convert2rhel_facts['conversions.env.CONVERT2RHEL_THROUGH_FOREMAN'] == '1'
```

Comment:
Can you add a validation for a `conversions.env.success` too as you're reading the whole file or along with any other facts that you feel could add a value here from C2R facts?

Reply:
There is "conversions.success": true but it is already asserted with facts so I don't want to duplicate it. The scope of the feature is covered in this but we can add "conversions.target_os.name": "Red Hat Enterprise Linux Server" assertion

Reply:
conversions.success we're reading on Satellite facts, can't we check this fact with satellite too instead of checking on host directly?

Reply:
We should check on host and optionally can check on Satellite too,

Reply:
anyway this facts are going to be uploaded on sat, so I feel its best to check it on sat instead of hos, but I leave this upto you

Reply:
Yes, True. Just for the sake of covering all, I am verifying on both